apps/server/src/modules/user/legacy/users-admin.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { LoggerModule } from '@src/core/logger';
+import { UserRepo } from '@shared/repo';
+import { AdminUsersService } from './service';
+import { UsersAdminRepo } from './repo';
+
+@Module({
+	imports: [LoggerModule],
+	providers: [UserRepo, AdminUsersService, UsersAdminRepo],
+	exports: [AdminUsersService],
+})
+export class UsersAdminModule {}
```

Comment:
The module and the repo are called "UsersAdmin..." and the services and UCs are called "AdminUsers..." (sometimes with "Api" in the middle). Is there a reason for these differences? Otherwise I would say: stick to one naming pattern!